js/functions.js

```diff
@@ -0,0 +1,43 @@
+//Функция для проверки, является ли строка палиндромом
+
+const checkIfPalyndrome = (string) => {
+  string=string.replaceAll(' ', '');
+  string=string.toLowerCase();
+  let newString = '';
+  for (let i=string.length-1; i>=0;i--) {
+  newString += string[i];}
+  return newString === string ?  true : false;
+  }
+
```

Comment:
обычно так не делают, можно сразу результат сравнения возвращать, он как раз будет логического типа
```suggestion
  return newString === string;
  }

```